src/lib/utils/addressUtils.ts

```diff
@@ -0,0 +1,15 @@
+const ZERO_ADDRESS_SHORTHAND_REGEX = /^0x0$/;
+const ZERO_ADDRESS_SHORTHAND_SEARCH_REGEX = /'0x0'/g;
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+export function extendZeroAddressShorthand(value) {
+  if (typeof value === 'string' && value.match(ZERO_ADDRESS_SHORTHAND_REGEX) !== null) {
```

Comment:
I guess we cannot be sure it is a string and we cannot have a type `Maybe<string>`

Reply:
I've now changed this so that it always expects a string, and we ensure a string is passed to it.